puppet/modules/slave/manifests/packaging/rpm.pp

```diff
@@ -71,7 +72,7 @@
     source => 'puppet:///modules/slave/katello-ca.cert',
   }
 
-  unless $is_el8 {
+  unless $is_el8 or $is_el9 {
     # tito
```

Comment:
Can't we drop tito by now?

Reply:
Maybe, I think @Odilhao has merged all the Pulpcore PRs to switch away from tito.

Reply:
Yes, all PR's are now merged.

Reply:
https://github.com/theforeman/foreman-infra/pull/1895